README.md

```diff
@@ -250,7 +278,23 @@ will take precedence.
 
 Hyrax is available under [the Apache 2.0 license](LICENSE.md).
 
-# Contributing
+# Quick Start for Hyrax development and contributing code back
```

Comment:
@elrayle I'm having trouble picking out the specific reordering that helps uses avoid the PT error, because there are so many changes here (adding the quick start section, adding the dev section, renaming other sections). It would help me review this if you could split these out, with one commit making the smallest set of changes to avoid the PT error, and another commit with the other changes. Sorry to be a pain -- I keep a very close eye on the README!